.github/workflows/ci.yaml

```diff
@@ -13,13 +13,12 @@ on:
     branches:
     - master
   pull_request:
-    branches:
-    - master
   schedule:
     - cron: "0 5 * * TUE"
 
 env:
-  CACHE_NUMBER: 1 # Change this value to manually reset the environment cache
+  CONDA_CACHE_NUMBER: 1 # Change this value to manually reset the environment cache
+  BUNDLE_CACHE_NUMBER: 1
```

Comment:
`BUNDLE_CACHE_NUMBER` is not used. Should be used in a similar way as the `CONDA_CACHE_NUMBER` to manually trigger a rebuild of the databundle cache when changed.

Reply:
You're right. Not sure how is missed it.